graph-courses.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+import argparse
+import collections
+import operator
+import calendar as cal
+from Class import *
+from ConvertParse import *
+
+reload(sys)
```

Comment:
Again, we shouldn't need to play any games with the default encoding here. If we run into encoding problems, let's solve them explicitly.